src/pages/SearchPage.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from '../components/Button'
+import CropItem from '../components/CropItem'
+import Header from '../components/Header'
+
+SearchPage.propTypes = {
+  crops: PropTypes.array.isRequired,
+  onClickDetails: PropTypes.func.isRequired,
+  onClickFavorites: PropTypes.func.isRequired,
+}
+
+export default function SearchPage({
+  crops,
+  onClickFavorites,
+  onClickDetails,
+}) {
+  const [searchTerm, setSearchTerm] = useState('')
+  const [searchResults, setSearchResults] = useState([])
+
+  return (
+    <Wrapper>
+      <Header>Search</Header>
+      <Input
+        placeholder="Search your favorite crops  🔍"
```

Comment:
just a comment on the text: do you want to say "nach Früchten suchen" or "Früchte durchsuchen"?
I think "Search for your favorite crops" would be better (But I can be completely wrong here)